tests/misc/projects/Issue8019/compile-fail.hxml.stderr

```diff
@@ -0,0 +1,20 @@
+Macro.hx:8: characters 17-18 : "" is not a valid package name:
+Macro.hx:8: characters 17-18 : Package name must not be empty
+Macro.hx:8: characters 17-18 : "\n" is not a valid package name:
+Macro.hx:8: characters 17-18 : Package name must start with a lower case character
+Macro.hx:8: characters 17-18 : "pack\n" is not a valid package name:
+Macro.hx:8: characters 17-18 : Invalid character: \n
+Macro.hx:8: characters 17-18 : "pack~" is not a valid package name:
+Macro.hx:8: characters 17-18 : Invalid character: ~
+Macro.hx:8: characters 17-18 : "Foo" is not a valid package name:
+Macro.hx:8: characters 17-18 : Package name must start with a lower case character
+Macro.hx:8: characters 17-18 : "0_class" is not a valid package name:
+Macro.hx:8: characters 17-18 : Package name must start with a lower case character
+Macro.hx:8: characters 17-18 : "0_enum" is not a valid package name:
+Macro.hx:8: characters 17-18 : Package name must start with a lower case character
+Macro.hx:8: characters 17-18 : "0_structure" is not a valid package name:
+Macro.hx:8: characters 17-18 : Package name must start with a lower case character
+Macro.hx:8: characters 17-18 : "0_abstract" is not a valid package name:
+Macro.hx:8: characters 17-18 : Package name must start with a lower case character
+Macro.hx:8: characters 17-18 : "0_abstract._Test" is not a valid package name:
```

Comment:
It's a bit weird that the error for the impl class shows up too, but not sure if that's worth worrying about.

Reply:
I think this check should skip `KAbstractImpl` classes.

Reply:
Aren't packages starting with "_" reserved by the compiler?
If yes, then this check could skip packages starting with underscore followed by an uppercase character.

Reply:
No, they're not reserved. Packages starting with `_` are considered "hidden" (don't show up in completion). Several libs like OpenFL take advantage of this.